SU-HSDO/suhumsci
# SHS-5691: New color palette for Traditional

# READY FOR REVIEW

## Summary
Adds a new color palette for Traditional

## Need Review By (Date)
08/23

## Urgency
medium

## Steps to Test
1. Go to https://hs-traditional.suhumsci.loc/admin/appearance/settings/humsci_traditional
2. Check there's a new `Color Pairing` that is called 'Vireo' and select it
3. Explore the whole site (Views, Components, Homepage, Mixed pages)
4. Check everything looks good and similar to what's in the [design](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=1532-1468&t=dNtrb9yDISRi2SEc-4)
5. Please also check the footer and header looks as good when the dark variation is selected

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208108045225555
